cmd/kpod/main.go

```diff
@@ -7,14 +7,17 @@ import (
 	"github.com/urfave/cli"
 )
 
+const VERSION string = "0.0.1"
```

Comment:
The tutorial I used for Go stated that it was good practice to make constants upper case, but I looked around and it can be camel case. Would you prefer I change it to be capitalized instead?